packages/components/pie-toast/src/index.ts

```diff
@@ -219,30 +236,78 @@ export class PieToast extends RtlMixin(LitElement) implements ToastProps {
         dispatchCustomEvent(this, ON_TOAST_CLOSE_EVENT, { targetNotification: this });
     }
 
+    /**
+     * Util method that returns an icon from a variant that has default icon.
+     *
+     * @private
+     */
+    private getVariantIcon () {
+        switch (this.variant) {
+            case 'info':
+                return html`<icon-info-circle class="icon" size="s" data-test-id="${componentSelector}-heading-icon-info"></icon-info-circle>`;
+            case 'success':
+                return html`<icon-check-circle class="icon" size="s" data-test-id="${componentSelector}-heading-icon-success"></icon-check-circle>`;
+            case 'warning':
+                return html`<icon-alert-triangle class="icon" size="s" data-test-id="${componentSelector}-heading-icon-warning"></icon-alert-triangle>`;
+            case 'error':
+                return html`<icon-alert-circle class="icon" size="s" data-test-id="${componentSelector}-heading-icon-error"></icon-alert-circle>`;
+            default:
+                return nothing as never;
+        }
+    }
+
+    /**
+     * Util method that returns true if the variant has an icon.
+     *
+     * @param {typeof variants[number]} variant - The variant to check if has icon.
+     *
+     * @private
+     */
+    private variantHasIcon (variant: typeof variants[number]) {
+        return ['info', 'success', 'warning', 'error'].includes(variant);
+    }
+
+    /**
+     * util method that returns true if the variant is warning and is strong.
+     *
+     * @private
+     */
+    private isVariantWarningAndIsStrong () {
+        return this.variant === 'warning' && this.isStrong;
+    }
+
     render () {
         const {
             isOpen,
+            variant,
             message,
             isDismissible,
             leadingAction,
             isMultiline,
+            isStrong,
             _messageAreaMaxWidth,
         } = this;
 
         if (!isOpen) {
             return nothing;
         }
 
+        const componentWrapperClasses = {
+            [componentClass]: true,
+            [`${componentClass}--variant-${variant}`]: true,
+            [`${componentClass}--is-strong`]: isStrong,
```

Comment:
Depending on how you update the variant class, will look something like this

```suggestion
            [`${componentClass}--${variant}`]: true,
            [`${componentClass}--strong`]: isStrong,
```